README.md

```diff
@@ -3,17 +3,19 @@ services: app-service, key-vault
 platforms: python
 author: lmazuel
 ---
-
 # Use Key Vault from App Service with Managed Service Identity and Python
 
+## SDK Versions
```

Comment:
See my comment on other keyvault sample and use the same format for SDK versions.